packages/react-components/src/components/VideoTile.tsx

```diff
@@ -151,7 +163,47 @@ const DefaultPlaceholder = (props: CustomAvatarOptions): JSX.Element => {
   );
 };
 
+/* @conditional-compile-remove(pinned-participants) */
+/**
+ * @internal
```

Comment:
@beta
This is a public facing interface

Reply:
done